test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -214,6 +212,8 @@ string TestFunctionCall::format(
 					stream << std::endl;
 			}
 		}
+
+		stream << formatGasExpectations(_linePrefix, _renderMode == RenderMode::ExpectedValuesActualGas, _interactivePrint);
```

Comment:
This looks like a leftover from the other PR. Is it needed here?

Reply:
I just changed the order here, that the gas expectations are only written after the side-effects. Basically, side-effects can only be parsed after a function call. This change makes that sure.

Reply:
Ah, so it's intentional. OK then.